app/models/concerns/uuidable.rb

```diff
@@ -1,13 +1,6 @@
 module Uuidable
   extend ActiveSupport::Concern
   included do
-    before_save :uuid
-
-    alias_method :generate_uuid!, :uuid
+    attribute :uuid, default: -> { SecureRandom.uuid }
```

Comment:
Simplifies things slightly. I had hoped this would avoid the need to invoke callbacks on importing records, but it seems that there is an implicit callback somewhere that we stil rely on.